### DISCUSSION TITLE
搶元宇宙商機，群創首度亮相裸眼 N3D 顯示技術

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#180] [General]
又又又是一個關於元宇宙的AI相關新聞ㄟ~~
https://ccc.technews.tw/2022/01/03/vr-n3d/

**Comment by cobb110202008** [comment]
元宇宙的話題熱度之高，顯現了現代科技對於虛擬實境及人工智慧的重點發展。

**Comment by briankuo123** [comment]
目前沒有很看好元宇宙，我認為他們目前做出來的東西沒有甚麼有趣的還不足以讓我買單。

**Comment by wh20502** [comment]
我覺得還遠，這些3d可能戴眼鏡比較實際，因為光學設計的緣故，裸眼要克服很多問題
